src/core/AndroidConvertor.js

```diff
@@ -0,0 +1,108 @@
+import BaseConvertor from './BaseConvertor'
+class AndroidConvertor extends BaseConvertor {
+
+    translateFromConversionItems(args) {
+        let availableLang = this._getAvailableLang(args);
+        let groupedItems = this._groupBy(args);
+        let groupedKey = Object.keys(groupedItems);
+
+        let stringXML = '<?xml version="1.0" encoding="utf-8"?> \n  <!-- generation time : ' + new Date().toISOString() + '--> \n<resources>\n';
+
+        groupedKey.forEach(key => {
+            stringXML += key.trim().length == 0 ? "\n" : `<!-- ${key} -->\n`;
+            stringXML += this._printGroup(groupedItems[key], availableLang[0]);
+        });
+        stringXML += '</resources>'
+        return stringXML;
+    }
+
+    _getAvailableLang(convertionItems) {
+        let lang = [];
+        convertionItems.forEach(element => {
+            let langKeys = Object.keys(element.values);
+            for (let index = 0; index < langKeys.length; index++) {
+                if (lang.indexOf(langKeys[index]) == -1) {
+                    lang.push(langKeys[index]);
+                }
+            }
+        });
+        return lang;
+    }
+
+    _groupBy(convertionItems) {
+        let groupedBySectionItems = {};
+        convertionItems.forEach(element => {
+            let elementSection = element.meta[this._config["groupBy"]];
+            elementSection = typeof elementSection === "undefined" ? "" : elementSection;

```

Comment:
Si tu veux vérifier qu'elementSection a une valeur et sinon lui assigner `""`, tu peux faire: 
`let elementSection = element.meta[this._config["groupBy"]] || "";`